packages/core/src/browser/core-preferences.ts

```diff
@@ -232,6 +232,23 @@ export const corePreferenceSchema: PreferenceSchema = {
             type: 'boolean',
             default: false,
             description: nls.localize('theia/core/tabMaximize', 'Controls whether to maximize tabs on double click.')
+        },
+        'workbench.tab.shrinkToFit.enabled': {
+            type: 'boolean',
+            default: false,
+            description: nls.localize('theia/core/tabShrinkToFit', 'Shrink tabs to fit available space')
```

Comment:
minor: for consistency preferences usually include punctuation

```suggestion
            description: nls.localize('theia/core/tabShrinkToFit', 'Shrink tabs to fit available space.')
```

The same is true for other preferences introduced in the pr.

Reply:
Other preferences also use whole sentences in the description, where a period at the end makes sense.